netlogo-gui/src/main/app/AppTabManager.scala

```diff
@@ -771,5 +827,24 @@ class AppTabManager(val appTabsPanel:          Tabs,
     }
   }
 
+  def __countMenuItembyNameAndMenuName(menuName: String, menuItemName: String): Int = {
+    getMenuByName(menuName)  match {
+      case None                => return 0
```

Comment:
Either the `=>` should align with each-other (preferred), or the `None` case should be with one space, but it shouldn't be far out like this.